src/theme/sections/News.tsx

```diff
@@ -14,22 +14,19 @@
  * limitations under the License.
  */
 import React from 'react';
-import Section from '../components/Section';
 import Triangle from '../components/Triangle';
-import { PostContainer } from '../components/Post';
 import { SECTION } from '../utils/constants';
 import { newsContent } from '../../content/NewsContent';
+import { SectionWithPosts } from './SectionWithPosts';
 
-const News = (): JSX.Element => {
-  const { news } = newsContent;
-
-  return (
-    <Section.Container id={SECTION.news} Background={Background}>
-      <Section.Header name={SECTION.news} />
-      <PostContainer posts={news} />
-    </Section.Container>
-  );
-};
+const News = (): JSX.Element => (
+  <SectionWithPosts
+    containerTitle={SECTION.news}
+    Background={Background}
+    posts={newsContent.news}
+    pageId="news"
```

Comment:
⚠️ for now we have less than 7 news, so directing the 'news' page doesn't provide new content.
However, I don't think it's worth taking time to hide the 'More' button we we have less than 7 elements in the list. We should put energy on writing more news to drop the need for hiding the button 😈 
Feel free to merge without changing anything here.

Reply:
Done